templates/core/core-dpl.yaml

```diff
@@ -144,6 +144,9 @@ spec:
 {{- with .Values.core.extraEnvVars }}
 {{- toYaml . | nindent 10 }}
 {{- end }}
+        {{- if not (empty .Values.containerSecurityContext) }}
+        securityContext: {{ .Values.containerSecurityContext | toYaml | nindent 10 }}
+        {{- end }}
```

Comment:
@MinerYang How about the `core-pre-upgrader-job.yaml`, we may also need to apply the same change to that file as well.